app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::MessagesController < ApplicationController
+  def index
+    random_message = Message.order('RANDOM()').first
+    render json: { message: random_message.text }
+  end
+end
```

Comment:
- [ ] You might need to implement `cors` in your backend in order for your app to work in the frontend on another browser on different machine.

### App behaviou on frontend
 
![app-behave-1](https://github.com/Tekhlay/hello-rails-back-end/assets/26597432/94b29d37-6096-4657-90b8-8ff47f898aee)

![app-behave-2](https://github.com/Tekhlay/hello-rails-back-end/assets/26597432/8c1b070e-d0e2-473e-83cc-5b3dfb9e1bd9)